lib/cylc/task_job_mgr.py

```diff
@@ -41,7 +42,7 @@
 from cylc.hostuserutil import get_host, is_remote_host, is_remote_user
 from cylc.job_file import JobFileWriter
 from cylc.task_job_logs import (
-    JOB_LOG_JOB, get_task_job_log, get_task_job_job_log,
+    JOB_LOG_OPTS, JOB_LOG_JOB, get_task_job_log, get_task_job_job_log,
```

Comment:
Unused `JOB_LOG_OPTS` import?

Reply:
Yes - moved that to workflow level info.